src/common/select-objects.js

```diff
@@ -57,7 +57,10 @@ const getLabel = object =>
   predicate: propTypes.func,
   required: propTypes.bool,
   value: propTypes.any,
-  xoContainers: propTypes.array,
+  xoContainers: propTypes.oneOfType([
+    propTypes.array,
+    propTypes.object
```

Comment:
It happens a map of containers with keys was convenient to me, and I checked that the code used these containers as a "collection", so array appeared to be too restrictive. Do you see an Issue with that ?